.github/workflows/release.yml

```diff
@@ -49,9 +49,7 @@ jobs:
         files: build/*
         name: ${{ github.event.inputs.version }} - ${{ github.event.inputs.title }}
         tag_name: ${{ github.event.inputs.version }}
-    - name: Commit CHANGELOG and Constants update
-      uses: stefanzweifel/git-auto-commit-action@v4
+    - name: Create pull-request for release changes
+      uses: peter-evans/create-pull-request@v3
```

Comment:
how does this action work? Does it auto-commit all the staged and unstaged changes? Or do we have to make the commit ourselves first?

Reply:
Also, I think the action allows to define the body of the PR, should we add something there?

Reply:
@fortmarek From the example, the actions seems to update with its own commit of unstaged files. 

@danyf90 Sure we can totally update the title of the commit, we should probably include the version and name, anyone have an actual string we should use?

Reply:
I think something like `[Release] Tuist X.X.X - ASD` should be ok 👍

Reply:
Added the message!